src/db/schema/passwords.ts

```diff
@@ -1,8 +1,8 @@
-import { pgTable, serial, text } from "drizzle-orm/pg-core";
+import { sqliteTable, integer, text } from "drizzle-orm/sqlite-core";
 import { users } from "./users";
 
-export const passwords = pgTable("passwords", {
-  userId: serial("userId")
+export const passwords = sqliteTable("passwords", {
+  userId: integer("userId")
```

Comment:
Have to change the datatype here since sqlite supports less datatypes and as a result it does not support `serial` so using `integer` instead.